app/retail/templates/emails/shared_template_head.html

```diff
@@ -0,0 +1,24 @@
+{% comment %}
+  Copyright (C) 2020 Gitcoin Core
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published
+  by the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see <http://www.gnu.org/licenses/>.
+{% endcomment %}
+{% load i18n static %}
+<link rel="stylesheet" href="{% static "v2/css/lib/typography.css" %}" data-premailer="ignore">
+<style>
+  {% include './assets/css/gitcoin.css' %}
+  {% include './assets/css/town_square.css' %}
+  {% include './assets/css/activity_stream.css' %}
+</style>
+<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap" rel="stylesheet" data-premailer="ignore">
```

Comment:
you cant insert stylesheets in emails bc many mail clients dont support it. youll ned to include all of these inline

Reply:
https://www.litmus.com/blog/do-email-marketers-and-designers-still-need-to-inline-css/

Reply:
I tried "include"ing the Roboto font css file by downloading it and adding it in the emails folder. But then the Roboto font doesn't seem to load neither on the web preview, Gmail Web nor on Apple Mail. But letting it as it is now, it loads fine on all three of them. Can you please try actually sending the email and see how it looks?